cli/command/container/logs.go

```diff
@@ -38,6 +39,7 @@ func NewLogsCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.follow, "follow", "f", false, "Follow log output")
 	flags.StringVar(&opts.since, "since", "", "Show logs since timestamp (e.g. 2013-01-02T13:23:37) or relative (e.g. 42m for 42 minutes)")
+	flags.StringVar(&opts.until, "until", "", "Show logs before a timestamp (e.g. 2013-01-02T13:23:37) or relative (e.g. 42m for 42 minutes)")
```

Comment:
Can you add an API-version annotation on this flag? That makes sure the flag is automatically hidden if the client connects to an older daemon;

```go
flags.SetAnnotation("until", "version", []string{"1.30"})
```